Documentation/ceph-cluster-crd.md

```diff
@@ -71,6 +71,7 @@ For more details on the mons and when to choose a number other than `3`, see the
 - `resources`: [resources configuration settings](#cluster-wide-resources-configuration-settings)
 - `storage`: Storage selection and configuration that will be used across the cluster.  Note that these settings can be overridden for specific nodes.
   - `useAllNodes`: `true` or `false`, indicating if all nodes in the cluster should be used for storage according to the cluster level storage selection and configuration values.
+  - `topologyAware`: `true` or `false`, indicating whether Rook will look for and use topology/failure domain labels on Kubernetes nodes (e.g. "region" or "zone") as part of the CRUSH location for OSDs.
```

Comment:
Could you either add an example of the exact label needed or add a link to the K8s docs that explain the labels?